src/BlockID.h

```diff
@@ -459,6 +459,7 @@ enum ENUM_ITEM_ID : short
 	E_ITEM_GOLD_HORSE_ARMOR = 418,
 	E_ITEM_DIAMOND_HORSE_ARMOR = 419,
 	E_ITEM_LEAD = 420,
+	E_ITEM_LEASH = E_ITEM_LEAD,
```

Comment:
'cause I said so. `Leash` is a much better name than `lead`, because it's unambiguous. Lead can be also a metal, or a verb.